spec/grape_entity/entity_spec.rb

```diff
@@ -296,24 +296,13 @@ class Parent < Person
           expect(subject.exposures).to eq(name: {}, email: {})
         end
 
-        # the following 2 behaviors are testing because it is not most intuitive and could be confusing
-        context 'when  called from the parent class' do
-          it 'remove from parent and all child classes that have not locked down their attributes with an .exposures call' do
+        context 'when called from the parent class' do
+          it 'remove from parent and do not remove from child classes' do
             subject.expose :name, :email
             child_class = Class.new(subject)
             subject.unexpose :email
 
             expect(subject.exposures).to eq(name: {})
-            expect(child_class.exposures).to eq(name: {})
-          end
-
-          it 'remove from parent and do not remove from child classes that have locked down their attributes with an .exposures call' do
```

Comment:
I am confused, does this spec break now? I mean when you unexpose :email from the parent class, the child no longer has it?

In which case this at least needs an UPGRADING entry, although I think you want to track down the PR in which this was added and figure out why?